### PR TITLE
Updated ToS

### PR DESCRIPTION
**Pull request by cjkindel**
Added instructions for generating from .rtf with Pandoc

----
📚 Documentation preview 📚: https://griptape--213.org.readthedocs.build//213/

### CODE REVIEW

File: docs/griptape-cloud/terms-of-service.md [hunk 1]
```diff
@@ -1,74 +1,391 @@
...
+**GRIPTAPE CLOUD TERMS OF SERVICE**
+
+**LAST UPDATED: FEBRUARY 14, 2024**
+
+**BY CLICKING** "**I ACCEPT,**"**ACCESSING OUR USING GRIPTAPE INC'S
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:5]: Need a space between the closed quote \" after 'accept' and 'accessing'

File: docs/griptape-cloud/terms-of-service.md [hunk 2]
```diff
@@ -1,74 +1,391 @@
...
+("GRIPTAPE"OR "WE" SERVICE**, YOU ("CUSTOMER"OR "YOU" AGREE THAT YOU
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:6]: Need a space between the closed quote \" after 'customer' and 'or'

File: docs/griptape-cloud/terms-of-service.md [hunk 3]
```diff
@@ -1,74 +1,391 @@
...
+    available at and is licensed for use by Customer under an open
+    source license ("**Griptape Framework**". As part of the Service,
+    Griptape may make its API** **(as defined below) and SDK (as defined
+    below) available to Customer.** **The Service, API, and SDK are
+    collectively referred to herein as the "**Griptape IPR**"and any
+    references to the Griptape IPR includes all components and portions
+    thereof. Subject the terms and conditions of this Agreement,
+    Griptape will provide Customer with access the Service using access
+    credentials provided by Griptape to permit Customer ("**Access
+    Credentials**" to access and use the Service. Customer will provide,
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:41]: Caught my own typo, this parenthetical string is not needed at all. Can cut parentheses and all chars between them.

File: docs/griptape-cloud/terms-of-service.md [hunk 4]
```diff
@@ -1,74 +1,391 @@
...
+
+7.  **INTELLECTUAL PROPERTY.**
+
+    1.  **Ownership.** As between the parties, Griptape owns all right,
+        title, and interest, including all intellectual property rights,
+        in and to the Griptape IPR, including any improvements,
+        modifications, and enhancements to them. Except for those rights
+        expressly granted in this Agreement, no other rights to the
+        Griptape IPR are granted, either express or implied, to
+        Customer.** **
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:142]: I think there's a stray \** bold markdown here.

File: docs/griptape-cloud/terms-of-service.md [hunk 5]
```diff
@@ -1,74 +1,391 @@
...
+
+    2.  **Customer Data. **Customer hereby grants to Griptape a
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:144]: Bold markdown closure \** is out of place, should be right after 'Data' before period?

File: docs/griptape-cloud/terms-of-service.md [hunk 6]
```diff
@@ -1,74 +1,391 @@
...
+        royalty-free, fully paid up, non-exclusive, and worldwide
+        license (with the right to sublicense through multiple tiers),
+        during the Agreement Duration, to use, host, copy, store,
+        transmit, publicly display, modify, and create derivative works
+        of, the Customer Data (as defined below) solely for purposes
+        of: (a) providing the Griptape IPR; (b) developing, creating,
+        and/or generating the Analytics; and (c) improving, providing,
+        and enhancing the Griptape IPR.** **Customer owns and retains
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:152]: Another stray bold markdown? \**

File: docs/griptape-cloud/terms-of-service.md [hunk 7]
```diff
@@ -1,74 +1,391 @@
...
+        all right, title, and interest, including all related
+        intellectual property and proprietary rights, in and to the
+        Customer Data. As part of the Griptape IPR, Griptape will,
+        during the Agreement Duration, host Customer Data in Griptape's
+        cloud environment solely for purposes of providing the Griptape
+        IPR. Customer is solely responsible for its Customer Data
+        (including its content and accuracy, and for backing up Customer
+        Data). For purposes of this Agreement, "**Customer Data**"means
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:160]: Need space after closed quote \" and 'means'

File: docs/griptape-cloud/terms-of-service.md [hunk 8]
```diff
@@ -1,74 +1,391 @@
...
+        generated in the processing and use of Analytics, and any and
+        all new products, services, and developments, modifications,
+        customizations, or improvements to the Griptape IPR made based
+        on the Analytics. Customer will not interfere with the
+        collection of Analytics. Griptape will not disclose Analytics
+        externally, including in benchmarks or reports, unless such
+        Analytics has been (a) de-identified so that it does not
+        individually identify Customer or its users, or (b) aggregated
+        with analytics from other Griptape customers. For purposes of
+        this Agreement, "**Analytics**"means the data and information
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:186]: Needs space between closed quote \" and 'means'

File: docs/griptape-cloud/terms-of-service.md [hunk 9]
```diff
@@ -1,74 +1,391 @@
...
+    Customer Data using commercially reasonable administrative,
+    technical, and physical safeguards designed to protect against
+    illegal or unauthorized access or disclosure of such data. Please
+    read the Griptape's Privacy Policy (available at
+    https://www.griptape.ai/privacy-policy) carefully for information
+    relating to Griptape's collection, use, storage, and disclosure of
+    Customer's personal information.
+
+2.  **THIRD PARTY PRODUCTS**. For purposes of this Agreement, "**Third
+    Party Products**"means certain third-party applications, systems,
```
Comment (derekpai) [inline docs/griptape-cloud/terms-of-service.md:223]: Needs space between closed quote \" and 'means'

**Review by derekpai — CHANGES_REQUESTED** [review]
Some auto-markdown typos. Otherwise fine.

**Comment by collindutter** [comment]
I will fix the issue causing the build to fail

**Review by collindutter — APPROVED** [review]

**Review by derekpai — APPROVED** [review]
Looks good.

**Comment by derekpai** [comment]
Non-blocking question: In the Poor (not Rich) view, does it mean anything that the text in latter sections change color from white to blue after the first row? I don't think this affects anything, but odd.

**Pull request merged**